.github/workflows/_s3_publish.yml

```
@@ -0,0 +1,282 @@
+          done
+          
+          # Save uploads list for summary
+          echo -e "$UPLOADS_LIST" > uploads_list.txt
+
+      - name: Generate and upload release manifest
+        id: manifest
+        run: |
+          S3_PREFIX="${{ steps.upload.outputs.s3_prefix }}"
+          GIT_VERSION="${{ steps.version.outputs.git_version }}"
+          BUILD_DATE="${{ steps.version.outputs.build_date }}"
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)